epcis-context-protected.jsonld

```diff
@@ -21,7 +21,7 @@
         "@type": "xsd:string"
       },
       "action": {
-        "@type": "xsd:string"
```

Comment:
Although I agree that from a Linked Data perspective, it would be good to be able to map the string values of `action` ("ADD","OBSERVE","DELETE") to URIs with definitions, this is not quite as straightforward as it may seem because these three values don't have a single unambiguous meaning; their meaning depends on the type of EPCIS event in which they are used.  For example, within an `ObjectEvent`, "ADD" usually accompanies a `bizStep` value of `...commissioning`, and the attachment of an ID to a physical object.  In contrast, within an `AssociationEvent`, "ADD" has a different meaning, namely that the specified `childEPCs` have been aggregated to the `parentID`.  So yes, we can do this without causing any disruption to XML/JSON-only users, but the definitions for these will need to be contextual, for the 4 out of 5 EPCIS event types in which `action` appears.